### ISSUE TITLE
I had two errors  after 'git submodule update --init --recursive'

### ISSUE THREAD

**Issue by Llcm** [MarvinTeichmann/KittiSeg#78]
i had two errors after ''git submodule update --init --recursive'',

2017-07-07 19:02:03,842 ERROR Could not import the submodules.
2017-07-07 19:02:03,842 ERROR Please execute:'git submodule update --init --recursive'

i don't understand these problems because i had executed 'git submodule update --init --recursive',but it still give me two errors.

hope someone give me answers.
thanks!

**Comment by fastlater** [comment]
@Llcm  did you solved the problem? I am facing the same problem in windows

**Comment by Haoyan-Guan** [comment]
if you face the problem using the linux but not windows, you can try git clone https://github.com/MarvinTeichmann/KittiSeg.git instead of downloading the ZIP

**Comment by yuyijie1995** [comment]
Did you solved this problem?
I used below as .bat in windows
del %~dp0\incl\evaluation
del %~dp0\incl\seg_utils
del %~dp0\incl\tensorflow_fcn
del %~dp0\incl\tensorvision
mklink /D %~dp0\incl\evaluation %~dp0\submodules\evaluation
mklink /D %~dp0\incl\seg_utils %~dp0\submodules\evaluation\kitti_devkit
mklink /D %~dp0\incl\tensorflow_fcn %~dp0\submodules\tensorflow-fcn
mklink /D %~dp0\incl\tensorvision %~dp0\submodules\TensorVision\tensorvision
echo ok
pause

however when i python train.py , i met the same problem 

2018-10-08 13:43:46,036 INFO No environment variable 'TV_PLUGIN_DIR' found. Set to 'C:\Users\louxy126/tv-plugins'.
2018-10-08 13:43:46,036 INFO No environment variable 'TV_STEP_SHOW' found. Set to '50'.
2018-10-08 13:43:46,036 INFO No environment variable 'TV_STEP_EVAL' found. Set to '250'.
2018-10-08 13:43:46,036 INFO No environment variable 'TV_STEP_WRITE' found. Set to '1000'.
2018-10-08 13:43:46,036 INFO No environment variable 'TV_MAX_KEEP' found. Set to '10'.
2018-10-08 13:43:46,036 INFO No environment variable 'TV_STEP_STR' found. Set to 'Step {step}/{total_steps}: loss = {loss_value:.2f}; lr = {lr_value:.2e}; {sec_per_batch:.3f} sec (per Batch); {examples_per_sec:.1f} imgs/sec'.
2018-10-08 13:43:46,039 ERROR Could not import the submodules.
2018-10-08 13:43:46,039 ERROR Please execute:'git submodule update --init --recursive'

**Comment by louxy126** [comment]
Did you solved this problem?
I used below as .bat in windows
del %~dp0\incl\evaluation
del %~dp0\incl\seg_utils
del %~dp0\incl\tensorflow_fcn
del %~dp0\incl\tensorvision
mklink /D %~dp0\incl\evaluation %~dp0\submodules\evaluation
mklink /D %~dp0\incl\seg_utils %~dp0\submodules\evaluation\kitti_devkit
mklink /D %~dp0\incl\tensorflow_fcn %~dp0\submodules\tensorflow-fcn
mklink /D %~dp0\incl\tensorvision %~dp0\submodules\TensorVision\tensorvision
echo ok
pause

however when i python train.py , i met the same problem

2018-10-08 13:43:46,036 INFO No environment variable 'TV_PLUGIN_DIR' found. Set to 'C:\Users\louxy126/tv-plugins'.
2018-10-08 13:43:46,036 INFO No environment variable 'TV_STEP_SHOW' found. Set to '50'.
2018-10-08 13:43:46,036 INFO No environment variable 'TV_STEP_EVAL' found. Set to '250'.
2018-10-08 13:43:46,036 INFO No environment variable 'TV_STEP_WRITE' found. Set to '1000'.
2018-10-08 13:43:46,036 INFO No environment variable 'TV_MAX_KEEP' found. Set to '10'.
2018-10-08 13:43:46,036 INFO No environment variable 'TV_STEP_STR' found. Set to 'Step {step}/{total_steps}: loss = {loss_value:.2f}; lr = {lr_value:.2e}; {sec_per_batch:.3f} sec (per Batch); {examples_per_sec:.1f} imgs/sec'.
2018-10-08 13:43:46,039 ERROR Could not import the submodules.
2018-10-08 13:43:46,039 ERROR Please execute:'git submodule update --init --recursive'
